utils/intellisensePrepublish.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs');
```

Comment:
**suggestion:** How do you feel about moving this inside the `polaris-intellisense` directory?